test/unix/test_lwt_bytes.ml

```diff
@@ -530,4 +530,43 @@ let suite = suite "lwt_bytes" [
       >>= fun () ->
       Lwt.return check
     end;
+
+    test "bytes recv" begin fun () ->
+      let hostname = "127.0.0.1" in
+      let port = 20000 in
+      let get_socket_infos () =
+             Lwt_unix.gethostbyname hostname
```

Comment:
You should be able to use [`Unix.inet_addr_loopback`](https://caml.inria.fr/pub/docs/manual-ocaml/libref/Unix.html#VALinet_addr_loopback) instead of calling `gethostbyname`.